pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/bestpractices/xml/ApexAssertionsShouldIncludeMessage.xml

```diff
@@ -49,4 +49,71 @@ public class Foo {
 }
         ]]></code>
     </test-code>
+
+    <test-code>
+        <description>[apex] Support new Assert class (Apex v56.0) #4097</description>
+         <expected-problems>0</expected-problems>
```

Comment:
I'll add here one additional test case before I merge your PR to prove, that we now also detect Asserts without messages from the new class...